etc/kayobe/ansible/pulp-host-image-download.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Download an overcloud host image from Ark
+  hosts: localhost
+  vars:
+    # This var is an edited version of stackhpc_overcloud_host_image_url
+    # without the auth credentials in it. Auth is handled by username and
+    # password in the get_url task of this playbook
+    stackhpc_overcloud_host_image_url_no_auth: >-
+      {{ stackhpc_release_pulp_content_url }}/kayobe-images/
+      {{ openstack_release }}/{{ os_distribution }}/{{ os_release }}/
+      {{ 'ofed/' if stackhpc_overcloud_host_image_is_ofed else '' }}
+      {{ stackhpc_overcloud_host_image_version }}/overcloud-
+      {{ os_distribution }}-{{ os_release }}
+      {{ '-ofed' if stackhpc_overcloud_host_image_is_ofed else '' }}.qcow2
+
+  tasks:
+    - name: Print image information
+      debug:
+        msg: |
+          OS Distribution: {{ os_distribution }}
+          OS Release: {{ os_release }}
+          Image tag: {{ stackhpc_overcloud_host_image_version }}
+          OFED: {{ stackhpc_overcloud_host_image_is_ofed }}
+
+    # TODO: Add checksum support
+    - name: Download image artifact
+      get_url:
+        url: "{{ stackhpc_overcloud_host_image_url_no_auth }}"
+        username: "{{ stackhpc_release_pulp_username }}"
+        password: "{{ stackhpc_release_pulp_password }}"
+        unredirected_headers:
+          - "Authorization"
+        dest: "/tmp/{{ os_distribution }}-{{ os_release }}.qcow2"
+        owner: "{{ ansible_user }}"
+        group: "{{ ansible_user }}"
```

Comment:
```suggestion
        owner: "{{ ansible_facts.user }}"
        group: "{{ ansible_facts.user }}"
```

Reply:
Done

Reply:
This actually failed. I tried a bunch of other possible values but the default is to use the current user so may as well stick with that